code/modules/research/designs/machine_designs.dm

```diff
@@ -91,6 +91,16 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SCIENCE
 
+/datum/design/board/thermomachine/anomalous
+	name = "Anomalous Thermomachine Board"
+	desc = "The circuit board for an anomalous thermomachine."
+	id = "pyrothermomachine"
+	build_path = /obj/item/circuitboard/machine/thermomachine/anomalous
+	category = list(
+		RND_CATEGORY_MACHINE
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
```

Comment:
Can Engineering not print this?

Reply:
No, just as engineers cannot print anomaly neutralizers or bags of holding, I feel that they shouldn't be able to print anomaly related devices.

Reply:
How are they supposed to know to ask for one if they don't know the board even exists?

Reply:
That... is a very good point, I will add it to engineering